packages/aws-cdk/lib/cli.ts

```diff
@@ -90,7 +90,14 @@ async function parseCommandLineArguments() {
     .command('bootstrap [ENVIRONMENTS..]', 'Deploys the CDK toolkit stack into an AWS environment', (yargs: Argv) => yargs
       .option('bootstrap-bucket-name', { type: 'string', alias: ['b', 'toolkit-bucket-name'], desc: 'The name of the CDK toolkit bucket; bucket will be created and must not exist', default: undefined })
       .option('bootstrap-kms-key-id', { type: 'string', desc: 'AWS KMS master key ID used for the SSE-KMS encryption', default: undefined, conflicts: 'bootstrap-customer-key' })
-      .option('bootstrap-customer-key', { type: 'boolean', desc: 'Create a Customer Master Key (CMK) for the bootstrap bucket (you will be charged but can customize permissions, modern bootstrapping only)', default: undefined, conflicts: 'bootstrap-kms-key-id' })
+      .option('bootstrap-ecr-key-id', {
+        type: 'string',
+        desc: 'Encrypts the bootstrap ECR repository with an Customer Master Key (CMK) or with the key given by an optional key-Id' +
+          'Note: Changes to encryption settings of an existing repository will recreate it!',
```

Comment:
```suggestion
          'Note: Changes to encryption settings of an existing repository is not possible!' +
          'You must first manually delete the repository before changing the encryption settings.',
```

From testing, it is not possible to change this since we are using a hard coded resource name.